app/routes/schema/stable/$.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { type DataFunctionArgs, redirect } from '@remix-run/node'
+
+import { getEnvOrDieInProduction } from '~/lib/env.server'
+
+type TagObject = {
+  name: string
+}
+
+export async function loader({ params: { '*': path } }: DataFunctionArgs) {
+  const GITHUB_API_TOKEN = getEnvOrDieInProduction('GITHUB_API_ACCESS')
```

Comment:
```suggestion
  const GITHUB_API_TOKEN = getEnvOrDieInProduction('GITHUB_API_TOKEN')
```

Also, please document this environment variable in `contributing.mdx`.